### PR TITLE
Make user management commands public

### PR DESCRIPTION
**Pull request by cotsog**

### CODE REVIEW

File: user/enterprise/user-management.md [hunk 1]
```diff
@@ -0,0 +1,41 @@
...
+
+## Suspend and Unsuspend Users
+
+Suspend or unsuspend users. Builds triggered by suspended users are blocked by `travis-gatekeeper`.
+
+`travis suspend <login>` - suspend a user where `<login>` is the user's GitHub login.
+
+`travis unsuspend <login>` - unsuspend a user where `<login>` is the user's GitHub login.
+
+**Please note**: Using the `suspend` command does not restrict access to the Enterprise platform.
```
Comment (Pezi777) [inline user/enterprise/user-management.md]:
```suggestion
> **Please note** that using the `suspend` command does not restrict access to the Enterprise platform.
```

File: user/enterprise/user-management.md [hunk 2]
```diff
@@ -0,0 +1,41 @@
...
+It removes a seat restriction for an archived user. If a *suspended* user were to log in to the platform, the seat restriction would be imposed once more.
```
Comment (Pezi777) [inline user/enterprise/user-management.md]:
```suggestion
It removes a seat restriction for an archived user. If a *suspended* user logs into the platform, the seat restriction would be imposed once more.
```

File: user/enterprise/user-management.md [hunk 3]
```diff
@@ -0,0 +1,41 @@
...
+
+## What determines an active, inactive, and suspended user
```
Comment (Pezi777) [inline user/enterprise/user-management.md]:
```suggestion
## Difference between Active, Inactive, and Suspended Users
```

File: user/enterprise/user-management.md [hunk 4]
```diff
@@ -0,0 +1,41 @@
...
+
+* Active User: Is a user with a GitHub OAuth token and has not been marked as suspended.
```
Comment (Pezi777) [inline user/enterprise/user-management.md]:
```suggestion
* Active User: User with a GitHub OAuth token and not marked as suspended.
```

File: user/enterprise/user-management.md [hunk 5]
```diff
@@ -0,0 +1,41 @@
...
+* Inactive User: Is a user without a GitHub OAuth token and has not been marked as suspended.
```
Comment (Pezi777) [inline user/enterprise/user-management.md]:
```suggestion
* Inactive User: User without a GitHub OAuth token and not marked as suspended.
```

File: user/enterprise/user-management.md [hunk 6]
```diff
@@ -0,0 +1,41 @@
...
+* Suspended User: Is a user that has been marked as suspended. They may or may not have a GitHub OAuth token.
```
Comment (Pezi777) [inline user/enterprise/user-management.md]:
```suggestion
* Suspended User: User marked as suspended having a GitHub OAuth token or not.
```

File: user/enterprise/user-management.md [hunk 7]
```diff
@@ -0,0 +1,41 @@
...
+
+## Sync a User
```
Comment (Pezi777) [inline user/enterprise/user-management.md]:
```suggestion
## Syncing a User
```

File: user/enterprise/user-management.md (hunk 1 above)
Comment (cotsog) [reply user/enterprise/user-management.md]: This one I would keep to be consistent with the other "Please note" below.

Comment (Pezi777) [reply user/enterprise/user-management.md]: ok

**Comment by cotsog** [comment]
Thank you @Pezi777!

**Pull request merged**